module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaTrustedCertificateAuthority.md

```diff
@@ -22,40 +33,77 @@ Get-EntraBetaTrustedCertificateAuthority
 ```
 
 ## Description
-The Get-EntraBetaTrustedCertificateAuthority cmdlet gets the trusted certificate authority in Azure Active Directory (AD).
+
+The `Get-EntraBetaTrustedCertificateAuthority` cmdlet gets the trusted certificate authority in Microsoft Entra ID.
 
 ## Examples
 
 ### Example 1: Retrieve the trusted certificate authorities that are defined in your directory
+
+```powershell
+Connect-Entra -Scopes 'Organization.Read.All'
+Get-EntraBetaTrustedCertificateAuthority
```

Comment:
Confirm example works.

Getting this error:

![image](https://github.com/user-attachments/assets/3cd5e7d2-b112-466d-85b8-903bfdd16e5a)

Reply:
Yes example works properly for me.

![image](https://github.com/user-attachments/assets/e6598ebe-d8d7-41a5-87a0-fe9f49f207bf)